<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="windowModality">
   <enum>Qt::WindowModality::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>889</width>
    <height>650</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Dataset Tools</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="2">
    <widget class="QPushButton" name="generateCaptionsBtn">
     <property name="text">
      <string>Generate Captions</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QPushButton" name="openBtn">
     <property name="text">
      <string>Open</string>
     </property>
    </widget>
   </item>
   <item row="0" column="7">
    <widget class="QPushButton" name="helpBtn">
     <property name="text">
      <string>Help</string>
     </property>
    </widget>
   </item>
   <item row="0" column="5" colspan="2">
    <widget class="QCheckBox" name="includeSubdirCbx">
     <property name="text">
      <string>Include Subdirectories</string>
     </property>
    </widget>
   </item>
   <item row="1" column="6">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Brush Alpha</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QPushButton" name="generateMaskBtn">
     <property name="text">
      <string>Generate Mask</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QPushButton" name="fillBtn">
     <property name="text">
      <string>Fill</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" rowspan="3" colspan="2">
    <widget class="QListView" name="listView"/>
   </item>
   <item row="1" column="2">
    <widget class="QPushButton" name="drawBtn">
     <property name="text">
      <string>Draw</string>
     </property>
    </widget>
   </item>
   <item row="1" column="5">
    <widget class="QCheckBox" name="enableMaskEditCbx">
     <property name="text">
      <string>Enable Mask Editing</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2" colspan="6">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>DRAW AREA</string>
     </property>
    </widget>
   </item>
   <item row="1" column="7">
    <widget class="QDoubleSpinBox" name="alphaSbx">
     <property name="maximum">
      <double>1.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.100000000000000</double>
     </property>
     <property name="value">
      <double>1.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="3" column="2" colspan="6">
    <widget class="QLineEdit" name="promptLed"/>
   </item>
   <item row="1" column="4">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>openBtn</tabstop>
  <tabstop>generateMaskBtn</tabstop>
  <tabstop>generateCaptionsBtn</tabstop>
  <tabstop>includeSubdirCbx</tabstop>
  <tabstop>listView</tabstop>
  <tabstop>enableMaskEditCbx</tabstop>
  <tabstop>alphaSbx</tabstop>
  <tabstop>drawBtn</tabstop>
  <tabstop>fillBtn</tabstop>
  <tabstop>promptLed</tabstop>
  <tabstop>helpBtn</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
